shared/js/browser-detector.js

```diff
@@ -1,37 +1,31 @@
 // We support IE 11+
 function isUnsupportedIE() {
   var isIE = navigator.userAgent.indexOf('MSIE') !== -1;
-  var IEVersion = navigator.appVersion.indexOf('Trident/');
-  var IEBelow8 = isIE && IEVersion < 8;
-
-  var IE8 = navigator.userAgent.match('MSIE 8.0;');
-  var IE9 = navigator.userAgent.match('MSIE 9.0;');
-  var IE10 = navigator.userAgent.match('MSIE 10.0;');
-
-  var unsupported = IEBelow8 || IE8 || IE9 || IE10;
+  var IEVersion = /MSIE (\d*)./g.exec(navigator.userAgent)[1];
```

Comment:
That [1] is a quirky behaviour of exec.  Wonder if it's worth a comment for future viewers?